integration/src/main/java/io/keploy/servlet/middleware.java

```diff
@@ -153,6 +148,12 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
             Map<String, Service.StrArr> headerMap = getResponseHeaderMap(responseWrapper);
             Service.HttpResp httpResp = builder.setStatusCode(responseWrapper.getStatus()).setBody(responseBody).putAllHeader(headerMap).build();
 
```

Comment:
Add a short comment that why you did this for future reference.

Reply:
Done